Pipelines/oat-pr.yml

```diff
@@ -50,6 +50,7 @@ extends:
           poolImage: MSSecurity-1ES-Ubuntu-2204
           poolOs: linux
           projectPath: 'OAT.Tests/OAT.Tests.csproj'
+          dotnetTestArgs: '-- --coverage --report-trx'
```

Comment:
At the moment, `dotnet test` is still VSTest oriented so we need to pass the arguments down to MTP. The way to do so is to use `-- ARGS`